### PR TITLE
Updating Authoring for roles

### PR DESCRIPTION
**Pull request by nebadr**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json [hunk 1]
```diff
@@ -7841,6 +7841,10 @@
         "endCharIndex": {
           "description": "The index within the utterance where the extracted entity ends.",
           "type": "integer"
+        },
+        "role": {
+          "description": "The role.",
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: If possible, better be something more informative.

File: specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json [hunk 2]
```diff
@@ -7841,6 +7841,10 @@
         "endCharIndex": {
           "description": "The index within the utterance where the extracted entity ends.",
           "type": "integer"
+        },
+        "role": {
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: Please provide evidence of sign-off from Azure API review board.
Please add samples to cover the role.

**Review by yangyuan — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json (hunk 1 above)
Comment (nebadr) [reply specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: Addressed.

File: specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json (hunk 2 above)
Comment (kayousef) [reply specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: We are currently working with cognitive services on a unified process for APIs review and versioning as currently we don't have any unified pipeline.
Why is the Azure API review mandatory given that we are still working on the process? Also, what kind of evidence is required? can you elaborate?

Comment (nebadr) [reply specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: Mentioning @yangyuan

Comment (yangyuan) [reply specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: Every API design change (including adding new operations/parameters or changing of existing operations/parameters) requires review from Azure API Review Board.

Azure API Review Board can do online review (Teams meeting) or offline review (PR review). For the online review you can forward me the review email thread for the offline review they can just sign-off this PR.

LUIS team has done many reviews with Azure API review board, Carol should familiar with the process.

**Review by yangyuan — APPROVED** [review]

**Comment by kayousef** [comment]
@yaohaizh We got approval from the board and sent it to Yang Yuan. Please, remove the blocking label.

**Review by yaohaizh — APPROVED** [review]

**Pull request merged**